lib/gcov/Makefile.uk

```diff
@@ -0,0 +1,67 @@
+#  SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Shouldn't we use the same license header, with `Unikraft authors ...` and simplified content? @michpappas